src/index.js.flow

```diff
@@ -225,6 +238,13 @@ export type Keg = {
   tapDate: Date,
 };
 
+export type KegMutator = {
```

Comment:
you said that its probably should be also flow sensor id, but idk
there isn't that prop here: https://github.com/Brewskey/Brewskey.Web/blob/master/Source/Keg.Web/Models/Kegs/EditKeg.cs

I was trying 'flowSensorId'  -doesn't' work. and why do we need flowSensorId for setup keg at all?

Reply:
Yeah... I'll add the mutator class to the controller and that should fix things. I don't think the controller will work right now as is.